R/classdist.R

```diff
@@ -230,34 +230,40 @@ mah_pooled <- function(means, x, cov_mat) {
 
 #' @export
 bake.step_classdist <- function(object, new_data, ...) {
-  if (length(object$objects[[1]]$center) == 0) {
+  col_names <- names(object$objects[[1]][[1]])
```

Comment:
Transitioning to selection by position has me a little 🤨 but this looks safe.

Reply:
Yes, this wasn't ideal, but it is because the information is in two different places depending on `pool`. But it just happens to be that `names(object$objects[[1]][[1]])` captures both places 😄

Reply:
also, the positioning is just "first"